src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -148,4 +148,8 @@ public boolean equals(Object obj) {
                 && filteredPersons.equals(other.filteredPersons);
     }
 
+    @Override
+    public void updateViewList(String targetList) {
```

Comment:
Would you want this to be the list itself instead?

Reply:
err cos my show command class not maintaining any lists, it doesnt get any list from model, it only tells model to change the viewlist, so i just pass in a string that says "task", "event" or "reminder", then model will update the viewlist to whichever list is mentioned.

the view list will be called in the main window class to bind the anchor panes to show the list.